benchmark/catch2swar-demo.cpp

```diff
@@ -36,7 +36,9 @@ TEST_CASE("Atoi benchmarks", "[atoi][swar]") {
     REQUIRE(fromLIBC_STRLEN == fromZOO_NATURAL_STRLEN);
     REQUIRE(fromZOO_NATURAL_STRLEN == fromZOO_MANUAL_STRLEN);
     REQUIRE(fromGENERIC_GLIBC_STRLEN == fromZOO_NATURAL_STRLEN);
+#if ZOO_CONFIGURED_TO_USE_AVX
     REQUIRE(fromZOO_AVX == fromZOO_STRLEN);
+#endif
```

Comment:
I prefer to indent the conditional preprocessing with the code where it appears, it is a refinement, or more detail, with regards to the current level of indentation.  I don't know why colleagues overwhelmingly have the preference to flatten all conditional preprocessing.  This is especially nasty when the conditional preprocessing nests.